Sources/Fullscreen/Consent/Settings/ConsentViewCell.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+import UIKit
+
+public enum ConsentState: String {
+    case accepted = "På", withdrawn = "Av"
+}
+
+public struct ConsentViewCellModel {
+
+    public let title: String
+    public var state: ConsentState?
+    public let hairLine: Bool
+
+    public init(title: String, state: ConsentState? = nil, hairLine: Bool = true) {
+        self.title = title
+        self.state = state
+        self.hairLine = hairLine
+    }
+}
+
+public class ConsentViewCell: UITableViewCell {
+    static public let identifier = "consent-cell"
```

Comment:
I've seen that `UITableViewCell` conforms to  `Identifiable` protocol with `reuseIdentifier` static var.

Reply:
I'll look into that